.github/workflows/ci.yml

```diff
@@ -51,7 +52,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        browser: [chrome, firefox]
+        browser: [chrome]
```

Comment:
Something go wrong with Firefox testing?

Reply:
Firefox builds are erroring with

> Error: no DISPLAY environment variable specified

Haven't figured out why, will try and remedy before merge.

https://github.com/machty/ember-concurrency/actions/runs/7770826285/job/21191457704

Reply:
Restored ff testing; core issue was that I forgot there was a testem.js file that launched FF in headless mode that I forgot to move to test-app folder.